ONSdigital/eq-survey-runner
# EAR-1402-stocks

### What is the context of this PR?

This PR updates all of Stocks to the new versions inputted into Author.

The list of form types can be found on the Jira ticket.

### How to review 

1. Ensure that for every Stocks form type, there is a stocks_9*** variant.
2. Ensure that none of the updated schemas have a metadata named `employmentDate`
3. Ensure that all the eq_id's and form_type's are inputted correctly.
4. Ensure that the themes for the NI variants are inputted correctly.

### Checklist
